Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -583,3 +583,41 @@ def test_upload_entries_update_remote_system_command(mocker):
     mocker.patch.object(client, 'add_comment', side_effect=add_comment_request)
 
     update_remote_system_command(client, args, params)
+
+
+DATA = {'close_notes': 'This is closed', 'closed_at': '2020-10-29T13:19:07.345995+02:00', 'impact': '3',
+        'priority': '4', 'resolved_at': '2020-10-29T13:19:07.345995+02:00', 'severity': '1 - Low',
+        'short_description': 'Post parcel', 'sla_due': '0001-01-01T00:00:00Z', 'urgency': '3', 'state': '3',
```

Comment:
remove from here all the fields that aren't actually sent from the server, such as `state`
change `DATA` to an indicative name